squire_sdk/src/server/gathering/hall.rs

```diff
@@ -97,27 +99,31 @@ where
                     let sender = self.gatherings.get_mut(&id).unwrap();
                     let (send, recv) = oneshot_channel();
                     let msg = GatheringMessage::GetTournament(send);
-                    sender.send(msg);
+                    if !sender.send(msg) {
+                        panic!("GatheringMessage::GetTournament failed")
+                    }
```

Comment:
This is similar, maybe a gathering panicked or shut down. But this is a failure to write to disk. That's pretty bad, right?

Reply:
Similar to what I was saying before. A panic for `Permanent` actors should be reserved for errors that are completely unrecoverable. If we failed to get a copy of the tournament, that's fine. The tournament just shutdown. That should not stop the entire tournament management system.